pkg/volume/awsebs/aws_ebs.go

```diff
@@ -250,7 +250,7 @@ func getVolumeSource(
 
 func (plugin *awsElasticBlockStorePlugin) ConstructVolumeSpec(volName, mountPath string) (*volume.Spec, error) {
 	mounter := plugin.host.GetMounter(plugin.GetPluginName())
-	pluginDir := plugin.host.GetPluginDir(plugin.GetPluginName())
+	pluginDir := filepath.Join(plugin.host.GetPluginDir(plugin.GetPluginName()), util.MountsInGlobalPDPath)
```

Comment:
does it seem like enough plugins use this to warrant some new "GetPluginMountDir" interface?

Reply:
I'm not sure it would need to be an interface, rather just an exported function perhaps in `pkg/volume/util`?

```go
func GetPluginMountDir(host volume.VolumeHost, name string) string {
    pluginDir := filepath.Join(host.GetPluginDir(name), MountsInGlobalPDPath)
    return pluginDir
}
```

Something like that?

Reply:
This sounds reasonable to me